pkg/kwokctl/runtime/compose/cluster.go

```diff
@@ -504,7 +504,78 @@ func (c *Cluster) Down(ctx context.Context) error {
 	return nil
 }
 
-func (c *Cluster) Start(ctx context.Context, name string) error {
+func (c *Cluster) Start(ctx context.Context) error {
+	logger := log.FromContext(ctx)
+	conf, err := c.Config(ctx)
+	if err != nil {
+		return err
+	}
+
+	// TODO: nerdctl does not support 'compose start' in v1.1.0 or earlier
+	// Support in https://github.com/containerd/nerdctl/pull/1656 merge into the main branch, but there is no release
+	subcommand := "start"
+	if conf.Options.Runtime == RuntimeTypeNerdctl {
+		subcommand = "up"
+	}
+
+	commands, err := c.buildComposeCommands(ctx, []string{subcommand}...)
+	if err != nil {
+		return err
+	}
+
+	err = exec.Exec(ctx, c.Workdir(), exec.IOStreams{
+		ErrOut: os.Stderr,
+		Out:    os.Stderr,
+	}, commands[0], commands[1:]...)
+	if err != nil {
+		logger.Error("Failed to start cluster", err)
+	}
+
+	if conf.Options.Runtime == RuntimeTypeNerdctl {
+		err := c.SnapshotRestore(context.TODO(), path.Join(conf.Options.CacheDir, "restart.db"))
```

Comment:
```suggestion
		err := c.SnapshotRestore(ctx, c.GetWorkdirPath("restart.db"))
```

If there are multiple cluster backup files in `conf.Options.CacheDir` there will be conflicts and it should be placed in the respective working directory.

Reply:
@wzshiming Thanks for tips, updated